app/tests/services/nodeInfo.nodeinfo.test.js

```diff
@@ -0,0 +1,417 @@
+describe('NodeInfo Services', function() {
```

Comment:
hm, not sure about this test. IMO it would be better to test the `queryResultToObjects` factory
https://github.com/crate/crate-admin/blob/master/app/scripts/services/sql.js#L4-L19